js/adapt-contrib-xapi.js

```diff
@@ -19,7 +19,7 @@ define([
    * @callback ErrorOnlyCallback
    * @param {?Error} error
    */
-  var xAPI = Backbone.Model.extend({
+  var XAPI = Backbone.Model.extend({
```

Comment:
I'd prefer it if this was kept as xAPI.  Any other local variables can be renamed.